MekHQ/src/mekhq/campaign/parts/Refit.java

```diff
@@ -883,7 +883,7 @@ public void calculate() {
                 time = 40320;
             } else if (newEntity instanceof megamek.common.Dropship || newEntity instanceof megamek.common.Jumpship) {
                 time = 13440;
-            } else if (newEntity instanceof Mech || newEntity instanceof megamek.common.Aero || newEntity instanceof megamek.common.ConvFighter || newEntity instanceof megamek.common.SmallCraft) {
+            } else if (newEntity instanceof Mech || newEntity instanceof megamek.common.Aero) {
```

Comment:
Comment indicating it covers those two b/c Aero does.

Reply:
Honestly, I'd use the Entity.ETYPE_ flags here rather than type comparisons. Unless I'm operating under the false assumption that a type comparison is slower than an int comparison, in which case disregard.

Reply:
I believe instanceof is quite cheap.